desk/app/channels.hoon

```diff
@@ -1265,6 +1265,12 @@
   ::
       [=kind:c ship=@ name=@ rest=*]
     =/  =ship  (slav %p ship.pole)
+    ::
+    ::  ignore watch-acks for deleted channels
+    ?:  ?&  !(~(has by v-channels) kind.pole ship name.pole)
+            ?=(%watch-ack -.sign)
+        ==
+      cor
```

Comment:
Shouldn't we `%leave` in this case? Otherwise we'll still crash once we receive a `%fact`.

```hoon
(emit %pass pole %agent [src.bowl %channels-server] %leave ~)
```